### PR TITLE
document interactive use

### LINKED ISSUE #72: quick start with spark-shell
I like the way how https://github.com/LucaCanali/sparkMeasure#getting-started-examples-of-sparkmeasure-usage provide a code snippet for very quick start, what about something similar for silex?

It'd be easier, if it's in the maven central, but it's also possible with 'non-default' maven repos:

https://gist.github.com/Jiri-Kremser/efd968a3c8128fd125382da76b0f5b65

### PR DESCRIPTION
**Pull request by willb**
This PR shows how to use Silex from the sbt console or from a Spark shell.

Thanks to @Jiri-Kremser for the suggestion (and for documenting how to run Silex from a Spark shell).

Fixes #72.

### CODE REVIEW

File: src/jekyll/interactive.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+### Using a Spark shell
+
+You can run Silex under a Spark shell without explicitly downloading it.  Here's the invocation to make it work:
+
+{% highlight bash %}
+spark-shell --repositories https://dl.bintray.com/willb/maven/ --packages io.radanalytics:silex_2.11:{{ site.silexVersion }}
+{% endhighlight %}
+
+(Thanks to [Jirka Kremser](https://plus.google.com/+JiriKremser) for this suggestion!)
```
Comment (jkremser) [inline src/jekyll/interactive.md:17]: there is no need to mention me, it's up to you

**Comment by erikerlandson** [comment]
LGTM. The netlify link seems to be broken

File: src/jekyll/interactive.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+### Using the developer console
+
+If you've checked out the [Silex repository](https://github.com/radanalyticsio/silex), you can run `sbt console` and get an environment with Silex preloaded.  
+
+(In general, you can [customize the behavior of `sbt console`](https://chapeau.freevariable.com/2014/09/interactive-sbt.html).)
```
Comment (jkremser) [inline src/jekyll/interactive.md:23]: nice, I didn't know that. I did something similar for ruby before

**Comment by jkremser** [comment]
lgtm

**Pull request merged**